pkg/apis/networking/validation/validation_test.go

```diff
@@ -1228,6 +1233,13 @@ func TestValidateIngressCreate(t *testing.T) {
 			},
 			expectedErrs: field.ErrorList{field.Invalid(field.NewPath("spec.rules[0].http.paths[0].path"), "/([a-z0-9]*)[", "must be a valid regex")},
 		},
+		"backend.Resource field not allowed on create": {
+			tweakIngress: func(ingress *networking.Ingress) {
+				ingress.Spec.Backend = &networking.IngressBackend{
+					Resource: resourceBackend}
+			},
+			expectedErrs: field.ErrorList{field.Forbidden(field.NewPath("spec.backend.resource"), "cannot be set on this object")},
+		},
```

Comment:
additional unit tests:
* a path resource backend is not allowed on create
* a path resource backend and service name are not allowed together
* a path resource backend and service port are not allowed together
* a spec.backend resource and service name are not allowed together
* a spec.backend resource and service port are not allowed together